### ISSUE TITLE
While using sqlalchemy backend 301 redirect remained as QUEUED

### ISSUE THREAD

**Issue by RajatGoyal** [scrapinghub/frontera#68]
Only redirected url is marked marked as CRAWLED but the redirected remains as QUEUED.

**Comment by sibiryakov** [comment]
Yes, that's confusing. At the same time it depends on canonical solver behavior. So I think, the right way is to mark as CRAWLED url considered as canonical, and leave other urls canonical solver to decide.
